mobile/src/main/res/values/strings.xml

```diff
@@ -23,6 +24,15 @@
     <!-- Errors -->
     <string name="error_network">Oops, something went wrong. Please check your network connection and try again.</string>
     <string name="error_empty_vendor_menu">It looks like you have not yet added any menu items using the web interface. On Android you can only mark them as being in or out of stock.\n\nVisit the website at https://www.truckmuncher.com</string>
+    <string name="setting_item_unavailable_warning">setting_item_unavailable_warning</string>
+    <string name="setting_item_unavailable_warning_title">Item Unavailable Warning</string>
+    <string name="setting_item_unavailable_warning_summary">Show when entering serving mode</string>
+    <string name="dont_show_again">Don\'t show this message again</string>
+    <string name="items_unavailable_message">You have %s item(s) marked as unavailable. Would you like to double check your menu to review the items?</string>
+    <string name="items_unavailable_title">Items Unavailable</string>
+    <string name="items_unavailable_positive_button">Ignore</string>
```

Comment:
feels like this buttons are labeled wrong. The ignore should be the negative option since the prompt is to modify the menu